app/electron/__tests__/test-data/artifacthub-pkg.yml

```diff
@@ -0,0 +1,40 @@
+version: 0.1.0
+name: headlamp_minikube
+displayName: Minikube
+createdAt: '2024-11-08T10:00:00Z'
+logoURL: 'https://minikube.sigs.k8s.io/images/logo/logo.png'
+description: A Headlamp plugin for managing Minikube clusters
+annotations:
+  headlamp/plugin/archive-url: 'https://github.com/headlamp-k8s/plugins/releases/download/example-change-logo-0.0.1/change-logo-0.0.1.tar.gz'
+  headlamp/plugin/archive-checksum: 'SHA256:8a2a23d116fe175b4350ffc6cc259781ed98d943f932e30fc5c8fe4e07b86e89'
+  headlamp/plugin/version-compat: '>=0.30'
+  headlamp/plugin/distro-compat: 'in-cluster,web,docker-desktop,desktop'
+  headlamp/plugin/extra-files/0/url: 'https://github.com/kubernetes/minikube/releases/download/v1.35.0/minikube-linux-amd64.tar.gz'
+  headlamp/plugin/extra-files/0/checksum: 'SHA256:56da5a1a5265e2d1a315d08073e29f0793579b6bf82202cd1e297d95a54a3b2a'
+  headlamp/plugin/extra-files/0/arch: 'linux/x64'
+  headlamp/plugin/extra-files/0/output/minikube/output: 'minikube'
+  headlamp/plugin/extra-files/0/output/minikube/input: 'out/minikube-linux-amd64'
+  headlamp/plugin/extra-files/0/output/docker-machine-driver-kvm2/output: 'docker-machine-driver-kvm2'
+  headlamp/plugin/extra-files/0/output/docker-machine-driver-kvm2/input: 'out/docker-machine-driver-kvm2'
+  headlamp/plugin/extra-files/1/url: 'https://github.com/kubernetes/minikube/releases/download/v1.35.0/minikube-linux-arm64.tar.gz'
+  headlamp/plugin/extra-files/1/checksum: 'SHA256:c87e39b6e408c9b156bdd87f64a4d83c0d7c3901ddf1fc5cbced60a9c203fbf7'
+  headlamp/plugin/extra-files/1/arch: 'linux/arm64'
+  headlamp/plugin/extra-files/1/output/minikube/output: 'minikube'
+  headlamp/plugin/extra-files/1/output/minikube/input: 'out/minikube-linux-arm64'
+  headlamp/plugin/extra-files/2/url: 'https://github.com/kubernetes/minikube/releases/download/v1.35.0/minikube-darwin-amd64.tar.gz'
+  headlamp/plugin/extra-files/2/checksum: 'SHA256:e2eadd14acaeb9876b70ed2b9f1ba3931125ec4b63e166b1088abece119494ea'
+  headlamp/plugin/extra-files/2/arch: 'darwin/x64'
+  headlamp/plugin/extra-files/2/output/minikube/output: 'minikube'
+  headlamp/plugin/extra-files/2/output/minikube/input: 'out/minikube-darwin-amd64'
+  headlamp/plugin/extra-files/2/output/docker-machine-driver-hyperkit/output: 'docker-machine-driver-hyperkit'
+  headlamp/plugin/extra-files/2/output/docker-machine-driver-hyperkit/input: 'out/docker-machine-driver-hyperkit'
+  headlamp/plugin/extra-files/3/url: 'https://github.com/kubernetes/minikube/releases/download/v1.35.0/minikube-darwin-arm64.tar.gz'
+  headlamp/plugin/extra-files/3/checksum: 'SHA256:05880f5b488215d865f6a391c5b52da4b84b220e53b47b333f290ce1edaa4189'
```

Comment:
IMHO We should use yaml arrays instead of numbers

Reply:
Artifacthub only allows strings here, not arrays.

Reply:
other projects have used arrays and objects too

https://github.com/artifacthub/hub/blob/master/docs/olm_annotations.md
https://github.com/artifacthub/hub/blob/master/docs/krew_annotations.md

Reply:
Those do not use annotations? Note, they are not under "annotations".